cms/core/blocks/section_blocks.py

```diff
@@ -37,23 +37,14 @@ class Meta:
 
 
 class SectionBlock(StructBlock):
-    """The analysis section block definition."""
+    """The core section block definition with headers."""
 
     title = HeadingBlock()
     content = SectionContentBlock()
 
+    class Meta:
+        template = "templates/components/streamfield/section_with_header_block.html"
```

Comment:
```suggestion
        template = "templates/components/streamfield/section_block.html"
```

should be fine since a section is heading + content

Reply:
Fixed as part of [1ea7ed](https://github.com/ONSdigital/dis-wagtail/pull/57/commits/1ea7ed1e96e994acd51f46a43ee29a3956aa6c39)